src/main/java/RaiseTech/ControllerGet/NameController.java

```diff
@@ -0,0 +1,41 @@
+package RaiseTech.ControllerGet;
+
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+public  class NameController {
+    @GetMapping("/names")
+    public List<String> getNames() {
+        return List.of("koyama", "tanaka");
+    }
+
+
+    @PostMapping("/names")
+    public ResponseEntity<String> create(@RequestBody CreateForm form) {
+        // 登録処理は省略
+
+        URI url = UriComponentsBuilder.fromUriString("http://localhost:8080")
+                .path("/names/id") // id部分は実際に登録された際に発⾏したidを設定する
+                .build()
+                .toUri();
+        return ResponseEntity.created(url).body("name successfully created");
+    }
+
+    @PatchMapping("/names")
+    public ResponseEntity<Map<String, String>> update(@RequestBody CreateForm form) {
+        // 更新処理は省略
+        return ResponseEntity.ok(Map.of("message", "name successfully updated"));
+    }
+
+    @DeleteMapping("/names")
+    public ResponseEntity<Map<String, String>> delete(@RequestBody CreateForm form) {
```

Comment:
deleteの際はパスで削除対象のリソースのidを指定するようにすればいいと思います。
```suggestion
    @DeleteMapping("/names/{id}")
    public ResponseEntity<Map<String, String>> delete(@PathVariable int id) {
```
また、削除なのでFormは必要ないですね。